eng/tools/TypeSpecValidation/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "typespec-validation",
+    "version": "0.0.1",
+    "type": "module",
+    "main": "dist/TypeSpecValidation.js",
+    "bin": {
+        "tsv": "dist/TypeSpecValidation.js"
+    },
+    "dependencies": {
+        "simple-git": "^3.16.0"
+    },
+    "scripts": {
+        "prepare": "cd ${PWD} && npx tsc"
```

Comment:
it should be prepare and not postinstall because the root package.json won't link to the compiled js file, and therefore won't have the executable for tsv command

Reply:
I was thinking this should live in the root package.json, not in the TypeSpecValidation package.json.  The reason is that the TypeSpecValidation package itself is correct.  But because it's being consumed from sources, it needs to be compiled first.